qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/presenters/CreateCustomerPresenter.groovy

```diff
@@ -43,7 +45,17 @@ class CreateCustomerPresenter implements CreateCustomerOutput{
 
     @Override
     void customerCreated(String message) {
-        viewModel.successNotifications.add(message)
-        clearCustomerData()
+        try {
+            viewModel.successNotifications.add(message)
+            clearCustomerData()
+        } catch (MethodException listenerMethodException) {
+            //fixme
+            // Invocation of method selectionChange failed for `null`
+            // See https://github.com/qbicsoftware/qoffer-2-portlet/issues/208
+            log.error("Issue #208 $listenerMethodException.message")
```

Comment:
I don't know whether I should remove this again since it seems to be fixed now.

Reply:
i think you can keep it for reference.